CRM/Dedupe/Merger.php

```diff
@@ -919,22 +919,16 @@ public static function merge($dupePairs = [], $cacheParams = [], $mode = 'safe',
    *  An empty array to be filed with conflict information.
    *
    * @return bool
+   *
+   * @throws \CRM_Core_Exception
+   * @throws \CiviCRM_API3_Exception
+   * @throws \API_Exception
    */
   public static function skipMerge($mainId, $otherId, &$migrationInfo, $mode = 'safe', &$conflicts = []) {
 
     $conflicts = self::getConflicts($migrationInfo, $mainId, $otherId, $mode);
 
     if (!empty($conflicts)) {
-      foreach ($conflicts as $key => $val) {
```

Comment:
this is moved into skipMerge - basically is says 'if there are unresolved conflicts and we are in safe mode then skip, otherwise copy over resolution data, even if it is NULL'